Source/WarpX.H

```diff
@@ -491,6 +491,8 @@ public:
     const amrex::IntVect get_ng_depos_J() const {return guard_cells.ng_depos_J;}
     const amrex::IntVect get_ng_depos_rho() const {return guard_cells.ng_depos_rho;}
 
+    const amrex::IntVect get_numprocs() const {return numprocs;}
```

Comment:
```suggestion
    /** Coarses-level Domain Decomposition
     *
     * If specified, the domain will be chopped into the exact number
     * of pieces in each dimension as specified by this parameter.
     *
     * @return returns the number of MPI processes if specified, otherwise 0
     */
    const amrex::IntVect get_numprocs() const {return numprocs;}
```